readme.md

```diff
@@ -162,13 +198,13 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+> There's no ending carrot for the !DOCTYPE html tag or the closing title tag.
```

Comment:
Hah, you eat a carrot, you write a 'caret'!

There's one more thing that will prevent the actual content of the page from showing...